Program.cs

```diff
@@ -11,22 +11,17 @@ internal class Program
         static void Main(string[] args)
         {
 
-            try
-            {
-                Console.WriteLine("Hello");
-            }
-            catch (ArgumentNullException)
-            {
-                Console.WriteLine("A");
-            }
-            catch (Exception)
-            {
-                Console.WriteLine("B");
-            }
-            finally
-            {
-                Console.WriteLine("C");
-            }
+           int n1, n2, resultado;
+
+            Console.WriteLine("Digite qualquer número: ");
+            n1 = int.Parse(Console.ReadLine());
+
+            Console.WriteLine("Digite o segundo número: ");
+            n2 = int.Parse(Console.ReadLine());
+
+            resultado = n1 + n2;
+            Console.WriteLine("O Resultado é: " + resultado);
```

Comment:
Utilizar interpolação de strings:
```suggestion
            Console.WriteLine($"O Resultado é: {resultado}");
```